Apps/WebClient/src/ClientApp/test/landing.test.ts

```diff
@@ -26,24 +28,16 @@ describe("Landing view", () => {
         maxDependentAge: 12,
     };
 
-    const configGetters = {
-        webClient: (): WebClientConfiguration => {
-            return a;
-        },
+    const options = new StoreOptionsStub();
+    options.modules.config.getters.webClient = (): WebClientConfiguration => {
+        return a;
```

Comment:
Can the variable `a` be renamed to be more descriptive?

Reply:
Sure it can! will rename it